SETUP_ARCH.md

```diff
@@ -0,0 +1,48 @@
+# Snowdrift setup instructions for Arch Linux
+
+## Installing stuff
+
+The Arch Wiki already has documentation on the individual components, so
+I'll just link to those sequentially.
+
+1.  [Install Haskell][1].
+2.  [Install PostgreSQL][2]. You only need to go through the
+    "installing" step. We have a script, `sdm`, that will take care of
+    setting up a user, and everything thereafter.
+3.  Run these commands:
+
+        git clone https://git.gnu.io/snowdrift/snowdrift.git
+        cd snowdrift
+        ln -s cabal.config.7.10 cabal.config
+        cabal sandbox init
+        cabal install -fdev
+        sdm init
+        cabal install -fdev --enable-tests
+        yesod devel
+
+The site should now be running on <http://localhost:3000>.
+
+If you get an error "command not found: sdm", or "command not found:
+yesod", when you try to run those respective commands, make sure that
+`.cabal-sandbox/bin` is in your path. Note that this is a relative path,
+not an absolute path.
+
+## Workflow
+
+`yesod devel` will stay running in one terminal while you do work
+elsewhere. It will rebuild the site whenever you change certain
+files. Often times, it won't detect changes. In which case, stop `yesod
+devel` by hitting `Return` a few times, then start it up again.
```

Comment:
That's not true at all. First, yesod devel *does* recompile when .hs files are changed, not only when templates are changed. I don't know why you thought otherwise.

Second, `cabal install` does a clean automatically; cabal clean and cabal build together are enough.

So, the backend files don't even need *anything* done, yesod devel recognizes them automatically. The only times it needs clean and build is because it doesn't recognize changes to *cassius* files or a few other cases like *static* files. And the only time cabal install is needed over cabal build would be if you make entirely new libraries or new dependencies or something.